packages/usehooks-ts/src/useLocalStorage/useLocalStorage.ts

```diff
@@ -145,7 +145,9 @@ export function useLocalStorage<T>(
   })
 
   useEffect(() => {
-    setStoredValue(readValue())
+    const value = readValue()
+    setValue(value)
+    setStoredValue(value)
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
setValue already sets stored value, isnt it?
maybe we can remove the eslint now and fill the deps as the logic changed?